dags/crash_summary.py

```diff
@@ -18,10 +18,16 @@
 
 dag = DAG('crash_summary', default_args=default_args, schedule_interval='@daily')
 
-crash_summary_view = EMRSparkOperator(
+crash_summary_view = MozDatabricksSubmitRunOperator(
     task_id="crash_summary_view",
     job_name="Crash Summary View",
-    instance_count=20,
+    instance_count=5,
+    max_instance_count=40,
```

Comment:
We _may_ want to configure this with less, but we can change it later if the nodes seem under utilized.